src/Context/SymfonyUnleashContext.php

```diff
@@ -62,7 +62,9 @@ public function getCurrentUserId(): ?string
             }
             $reflection = new ReflectionObject($user);
             $idProperty = $reflection->getProperty($this->userIdField);
-            $idProperty->setAccessible(true);
+            if (\PHP_VERSION_ID < 80100) {
```

Comment:
Please use an unqualified const name:

```suggestion
            if (PHP_VERSION_ID < 80100) {
```